packages/core/src/external/quest/fhir/organization.ts

```diff
@@ -0,0 +1,79 @@
+import { uuidv7 } from "@metriport/shared/util/uuid-v7";
+import { Address, ContactPoint, Identifier, Organization, Reference } from "@medplum/fhirtypes";
+import { ResponseDetail } from "../schema/response";
+import { getQuestDataSourceExtension } from "./shared";
+
+/**
+ * The organization that ordered the tests.
+ * @returns {Organization} FHIR resource
+ */
+export function getInsuranceOrganization(detail: ResponseDetail): Organization {
+  const identifier = getOrganizationIdentifier(detail);
+  const name = getOrganizationName(detail);
+  const address = getOrganizationAddress(detail);
+  const telecom = getOrganizationTelecom(detail);
+  const extension = [getQuestDataSourceExtension()];
+
+  return {
+    resourceType: "Organization",
+    id: uuidv7(),
+    ...(identifier ? { identifier } : {}),
+    ...(name ? { name } : {}),
+    ...(address ? { address } : {}),
+    ...(telecom ? { telecom } : {}),
+    extension,
+  };
+}
+
+export function getOrganizationReference(organization: Organization): Reference<Organization> {
+  return {
+    reference: `Organization/${organization.id}`,
+  };
+}
+
+function getOrganizationIdentifier(detail: ResponseDetail): Identifier[] | undefined {
+  if (!detail.orderingAccountNumber) return undefined;
+  return [
+    {
+      system: "http://hl7.org/fhir/sid/us-npi",
+      value: detail.orderingAccountNumber,
+    },
+  ];
+}
```

Comment:
i think this is straight up wrong, npis are 10 digits, and the ordering account number is 18 digits. this ordering account number is like a unique cxid quest uses to identify their customers

Reply:
Addressed by adding NPI and UPIN separately.

Reply:
@keshavsaharia We confirmed that the NPI + UPIN are for the practitioner right? im fairly certain we did but wanted to double check